### PR TITLE
Queue with stacks

### PR DESCRIPTION
**Pull request by devonhackley**

### CODE REVIEW

**Comment by devonhackley** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code-challenges/401challenges/src/main/java/challenges/QueueWithStacks/PsuedoQueue.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+           int top = this.stack2.pop();
+
+           while(this.stack2.getTop() != null ){
+               // grab all values from stack 2 and add to stack1(queue)
+               this.stack1.push(this.stack2.pop());
+           }
+           return top;
+       } catch (NullPointerException ex){
+            System.out.println("The stack is empty");
+            return -1;
```
Comment (evanslaton) [inline code-challenges/401challenges/src/main/java/challenges/QueueWithStacks/PsuedoQueue.java]: What if the value in the node that's being dequeued is `-1`/
